### PR TITLE
84202472 turn off input echo by default

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses issue #4589  

Since this is trying to minimize the writing to the audit file, it could not depend on a input in the IDF file so instead it is using environmental variable called "DisplayInputInAudit" that shows the input file in the audit file only when set to Yes.

### CODE REVIEW

**Comment by JasonGlazer** [comment]
@mjwitte Mike I fixed the spacing issues

**Comment by JasonGlazer** [comment]
@mjwitte Where is the EnergyPlus coding style defined so that I can be consistent with it?

**Comment by mjwitte** [comment]
@JasonGlazer As far as I know, the style info is not complete yet.  You have to work by example.  I have some VS screen shots that cover some of the settings to match up with the existing code.

File: src/EnergyPlus/DataSystemVariables.cc [hunk 1]
```diff
@@ -83,6 +83,7 @@ namespace DataSystemVariables {
 	//  each individual HVAC controller with all controller iterations
 
 	std::string const MinReportFrequencyEnvVar( "MINREPORTFREQUENCY" ); // environment var for reporting frequency.
+	std::string const cDisplayInputInAuditEnvVar( "DisplayInputInAudit" ); // environmental variable that enables the echoing of the input file into the audit file
```
Comment (Myoldmopar) [inline src/EnergyPlus/DataSystemVariables.cc]: I think we should keep a consistent ALL CAPS for environment variables.  On Linux (and I'm assuming Mac), the case sensitivity is important.

**Comment by Myoldmopar** [comment]
The Linux showing test failures is simply due to how the develop branch moved in the midst of these tests.  @DeadParrot's changes moved the baseline with eio changes and such.  It's not a problem.  When the environment variable is updated we should merge develop into this branch to make sure the baseline is appropriate.  At which point the CI should report all clean and this should be merged in. @JasonGlazer I really don't mind making the change if you can confirm that you are OK with the variable name change.

**Comment by JasonGlazer** [comment]
@Myoldmopar @mjwitte  I agree that the environmental variables should be consistent. If you look at the full source file you will see that for the existing environmental variables some are all caps and others are not. The ones that are related to "Display" and "Ignore" are in camel case while the design day ones and threads (and others) are in all caps. Should I go ahead and make them all ALL CAPS?

**Comment by Myoldmopar** [comment]
Bah...didn't notice that.  I'd say _don't_ go through and change them all.  Just make the new one all caps since that's the direction I'd like to move toward.

**Comment by EnergyArchmage** [comment]
I happen to be adding an environment variable or two as well to help control things with new HVAC Sizing Simulations.  So do I need to change to use all (ugly) caps ?

**Comment by Myoldmopar** [comment]
Yes, please.

**Comment by JasonGlazer** [comment]
Changed DisplayInputInAudit to DISPLAYINPUTINAUDIT

**Comment by Myoldmopar** [comment]
@JasonGlazer This looks good.  This didn't get the update to develop that changed the coverage threshold for warning, but it'll be good once merged in.  

After develop stabilizes later this morning I'll merge this in. Thanks.

**Pull request merged**